build.psm1

```diff
@@ -3173,7 +3173,7 @@ function New-TestPackage
     else
     {
         $null = New-Item -Path $Destination -ItemType Directory -Force
-        Write-Verbose -Message "Creating destination folder: $Destination"
+        Write-Verbose -message "Creating destination folder: $Destination"
```

Comment:
It seems the change is not related to the PR. Below too.

Reply:
```suggestion
        Write-Verbose -Message "Creating destination folder: $Destination"
```